File: Linguagem C#/soma_impares.cs
// See https://aka.ms/new-console-template for more information
using System.Globalization;
CultureInfo CI = CultureInfo.InvariantCulture;

Console.WriteLine("Digite dois numeros: ");
int a = int.Parse(Console.ReadLine());
int b = int.Parse(Console.ReadLine());
if (a > b) {
    int x = b;
    b = a;
    a = x;
}

int soma = 0;
for(int i = a+1; i < b; i++) {
    if (i % 2 != 0) {
        soma = soma + i;
    }
}
Console.WriteLine("SOMA DOS IMPARES = " + soma);

File: Linguagem C++/soma_impares.cpp
#include <iostream>

using namespace std;

int main(){


    int a,b,x,i,soma;

    cout << "Digite dois numeros: \n";
    cin >> a >> b;
    if (a > b) {
        x = a;
        a = b;
        b = x;
    }

    soma = 0;
    for(i = a+1; i < b; i++) {
        if (i%2 != 0) {
        soma = soma + i;
        }
    }
    cout << "Soma dos impares = " << soma;

    return 0;
}
